en.yml

```diff
@@ -687,8 +687,11 @@ en:
         location: "Location: %{owner}"
         your_account: Your account
       quizzes_configure_side_nav:
-        ad_offer_title: Unlock offers
-        ad_offer_description: 'Generate leads, link to special offers, and get likes for your Facebook page'
```

Comment:
we usually don't delete keys, just to be safe, in the case that they are used elsewhere in the app, and also we definitely don't delete them until the UI changes are in prod. Once this is merged it will automatically get picked up by the api, so sometimes they make it to prod before they should